### DISCUSSION TITLE
Make it possible to drag a storage container while using a scooter

### DISCUSSION THREAD

**Discussion by pubrrr** [FakeFishGames/Barotrauma#10502] [Ideas and suggestions]
* It is possible to grab another player and drag that player while using a scooter.
* It is _not_ possible to use a scooter and grab a storage container.
* It is possible to grab another player who holds a storage container while using a scooter.

This is not logical. IMO, you should be able to also grab a storage container while using a scooter.

**Comment by BiffW** [comment]
This can cause other issues like when you are using a scooter while holding a container on the secondary hand, your character will perform attack with the storage container while being pulled by the scooter. That can be ridiculous.
My idea to solve this is to define 2 new hot keys as "primary shoot" and "secondary shoot" and rename the current "Shoot" key as "Shoot both"

**Reply by pubrrr** [reply]
Just to clarify this: This is because - to use the scooter - I have to press the right and left mouse button? Doing so for the storage container means attacking?

Why doesn't this happen when you drag another player? (Apart from the fact that it would be pretty hilarious to attack with another player, lol). I'm lacking insight into how this is called in the code, but the difference is probably, that you don't have the other player "in your hand", i.e. it doesn't show up in the inventory bar at the bottom of the screen. How about enabling this "grabbed - but not in your hand" state also for other things like containers?
I would bet that it's possible to drag pets along as you can grab them.
